zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -226,14 +236,14 @@ where
         .collect::<Result<_, _>>()
 }
 
-pub(crate) fn count_outputs(
+pub(crate) fn spendable_notes_meta(
     conn: &rusqlite::Connection,
-    account: AccountId,
-    min_value: NonNegativeAmount,
-    exclude: &[ReceivedNoteId],
     protocol: ShieldedProtocol,
     chain_tip_height: BlockHeight,
-) -> Result<usize, rusqlite::Error> {
+    account: AccountId,
+    selector: &NoteFilter,
```

Comment:
Nit:
```suggestion
    filter: &NoteFilter,
```
(and similarly elsewhere)